components/ide/jetbrains/launcher/main.go

```diff
@@ -637,6 +642,25 @@ func handleSignal() {
 	log.Info("asked IDE to terminate")
 }
 
+func updatePlatformProperties(platformOptionsPath string, configDir string, systemDir string) error {
+	buffer, err := os.ReadFile(platformOptionsPath)
+	if err != nil {
+		return err
+	}
+
+	content := string(buffer)
+
+	newContent := strings.Join([]string{
+		content,
+		fmt.Sprintf("idea.config.path=%s", configDir),
```

Comment:
We could instead make it a part of vm options with`-Didea.config.path` and `-Didea.system.path`. Not sure about precedence.

Reply:
idea.properties have precedence as it's appended later https://github.com/JetBrains/intellij-community/blob/4a918183ef774c275cb12ec31e74c50c9d841ead/native/XPlatLauncher/src/lib.rs#L183-L200